compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -2700,8 +2707,8 @@ void t_go_generator::generate_service_server(t_service* tservice) {
     f_types_ << indent() << "func (p *" << serviceName
                << "Processor) Process(ctx context.Context, iprot, oprot thrift.TProtocol) (success bool, err "
                   "thrift.TException) {" << endl;
-    f_types_ << indent() << "  name, _, seqId, err := iprot.ReadMessageBegin(ctx)" << endl;
-    f_types_ << indent() << "  if err != nil { return false, err }" << endl;
+    f_types_ << indent() << "  name, _, seqId, err2 := iprot.ReadMessageBegin(ctx)" << endl;
```

Comment:
Note: all the compiler changes starting from this line are because in Process function we specified the returned error type to be `TException` instead of just `error`. I'm not sure why is that, if we can make another breaking change to `TProcessor` and `TProcessorFunction` these changes can be avoided.